### PR TITLE
GitHub template grammar/spelling fixes

### PR DESCRIPTION
**Pull request by GwendolenLynch**
|      Q       |   A
|------------- | -----------
| Type         | improvement
| BC Break     | no
| Fixed issues | n.a.

#### Summary

Grammar & spelling fixes for GitHub templates

### CODE REVIEW

**Review by Ocramius — APPROVED** [review]
Thansk for teh fiexs!

**Comment by Majkl578** [comment]
This shouldn't have been merged into 2.8.

**Comment by Ocramius** [comment]
Groan, my mistaek...

File: .github/ISSUE_TEMPLATE/BC_Break.md [hunk 1]
```diff
@@ -18,15 +18,15 @@ Before reporting a BC break, please consult the upgrading document to make sure
 
 #### Summary
 
-<!-- Provide a summary desciribing the problem you are experiencing. -->
+<!-- Provide a summary describing the problem you are experiencing. -->
 
-#### Previous behavior
+#### Previous behaviour
```
Comment (Majkl578) [inline .github/ISSUE_TEMPLATE/BC_Break.md]: :trollface:

**Comment by GawainLynch** [comment]
> This shouldn't have been merged into 2.8.

Ugh, my bad too for the choice of branch. I blame the lack of :beers:

**Comment by Majkl578** [comment]
Can you please send the same patch against master? We don't merge up from lower branches.

**Comment by GawainLynch** [comment]
On it!

**Pull request merged**